### ISSUE TITLE
LC0094 false positive?

### ISSUE THREAD

**Issue by jwikman** [StefanMaron/BusinessCentral.LinterCop#1184]
I'm trying to figure out the purpose of LC0094 that turned up in the pre-release...

I get this info on the `Rec` parameter of below code
```al
        Page.RunModal(Page::"MyPage", Rec);
```
`info LC0094: A method invoked on a record must not contain same variable in parameter list as the one on which the call was made.`

Is the purpose to detect scenarios like `Rec.MyFunction(Rec)`?

In that case, I believe I have a false positive scenario. 🙂

**Comment by jwikman** [comment]
I cannot see any value of this rule, based on all of the problems that it emits in my different repos...

I don't understand why code like `Rec.MyFunction(Rec)` should not be allowed. 🤔

**Comment by Arthurvdv** [comment]
@iwanscheidsnc thank you for looking into this, I've merged the PR which is now available in the pre-release of the LinterCop.

**Comment by Arthurvdv** [comment]
Hi, over the past months I’ve been putting a lot of time and care into the successor of BusinessCentral.LinterCop: ALCops (https://alcops.dev/). Because of that, this repository is no longer actively maintained.

I’m closing this issue, as it won’t be handled here anymore. If this is still relevant for you, please have a look at ALCops/Analyzers (https://github.com/ALCops/Analyzers).

To make the transition easier, you can also follow the LinterCop Migration Guide: https://alcops.dev/docs/lintercop-migration/

If you have any questions or run into issues during the transition, feel free to reach out through ALCops. Happy to help 🙂
